### ISSUE TITLE
accept-no-yields-doc appears to have no effect.

### ISSUE THREAD

**Issue by andrewbest-tri** [PyCQA/pylint#2624]
### Steps to reproduce
1. Create a method with correct documentation. Yield from the method. Do not document the yield.
2. Run pylint on the method. Load docparams. Disable yield errors, W9014 and W9013
3. Receive yield errors anyway.

### Current behavior

cat trivial.py
```python
def trivial(param1, param2):
    """Trivial method.

    Args:
        param1 (int): param 1.
        param2 (int): param 2.
    """
    print(param1, param2)
    yield 0
```
pylint --load-plugins=pylint.extensions.docparams trivial.py --accept-no-yields-doc=y

```
************* Module trivial
trivial.py:1:0: C0111: Missing module docstring (missing-docstring)
trivial.py:2:0: W9013: Missing yield documentation (missing-yield-doc)
trivial.py:2:0: W9014: Missing yield type documentation (missing-yield-type-doc)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

### Expected behavior
```
************* Module trivial
trivial.py:1:0: C0111: Missing module docstring (missing-docstring)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

### pylint --version output
```
pylint 2.2.2
astroid 2.1.0
Python 3.5.2 (default, Nov 12 2018, 13:43:14) 
[GCC 5.4.0 20160609]
```
and after checking out preview
```
pylint 2.2.2
astroid 2.1.0
Python 3.5.2 (default, Nov 12 2018, 13:43:14) 
[GCC 5.4.0 20160609]
```

**Comment by PCManticore** [comment]
Thank you @andrewbest-tri

**Comment by andrewbest-tri** [comment]
Update: 
    I believe it is also the case that accept-no-return-doc is not being properly parsed as well.

Example from above updated:

cat trivial.py
```
def trivial(param1, param2):
    """Trivial method.

    Args:
        param1 (int): param 1.
        param2 (int): param 2.
    """
    print(param1, param2)
    yield 0

def trivial2(param1, param2):
     """Trivial method.
 
     Args:
         param1 (int): param 1.
         param2 (int): param 2.
     """
     print(param1, param2)
     return 0
```
pylint --load-plugins=pylint.extensions.docparams trivial.py --accept-no-yields-doc=y --accept-no-return-doc=y

```
************* Module trivial
trivial.py:1:0: C0111: Missing module docstring (missing-docstring)
trivial.py:2:0: W9013: Missing yield documentation (missing-yield-doc)
trivial.py:2:0: W9014: Missing yield type documentation (missing-yield-type-doc)
trivial.py:12:0: W9011: Missing return documentation (missing-return-doc)
trivial.py:12:0: W9012: Missing return type documentation (missing-return-type-doc)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

### Expected behavior
```
************* Module trivial
trivial.py:1:0: C0111: Missing module docstring (missing-docstring)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

**Comment by AWhetter** [comment]
I think the documentation of these options is misleading. Really the right way to disable these warnings would be to disable the relevant message (eg `pylint --load-plugins=pylint.extensions.docparams trivial.py --disable=missing-yield-doc,missing-yield-type-doc`).
The options exist so that you can ignore the warning when the type of the docstring cannot be figured out. For example if you have a docstring that is only a summary line

```python
def def trivial(param1, param2):
    """Trivial method."""

```

**Comment by andrewbest-tri** [comment]
The suggestion of --disable=missing-yield-doc,missing-yield-type-doc works as expected. Thanks! This is great.

I now have the question you posed in your reply.... what do the accept-no-foo options actually do?

**Comment by AWhetter** [comment]
I mentioned that in the second half of my comment:

> The options exist so that you can ignore the warning when the type of the docstring cannot be figured out.
